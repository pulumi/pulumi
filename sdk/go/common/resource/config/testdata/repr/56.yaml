my:name:
- a
- b
- c
- - nested: value
