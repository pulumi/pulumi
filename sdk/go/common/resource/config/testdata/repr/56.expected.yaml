decrypt:
  my:name: '["a","b","c",[{"nested":"value"}]]'
paths:
  my:name:
    value:
    - a
    - b
    - c
    - - nested: value
    string: '["a","b","c",[{"nested":"value"}]]'
    redacted: '["a","b","c",[{"nested":"value"}]]'
    object: HRAADltdaW50ZXJmYWNlIHt9/4MCAQL/hAABEAAA/4D/hH0ABAZzdHJpbmcMAwABYQZzdHJpbmcMAwABYgZzdHJpbmcMAwABYw5bXWludGVyZmFjZSB7ff+EKAABF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAAc/4IZAAEGbmVzdGVkBnN0cmluZwwHAAV2YWx1ZQ==
    secure: false
    isObject: true
  my:name[0]:
    value: a
    string: a
    redacted: a
    object: DhAABnN0cmluZwwDAAFh
    secure: false
    isObject: false
  my:name[1]:
    value: b
    string: b
    redacted: b
    object: DhAABnN0cmluZwwDAAFi
    secure: false
    isObject: false
  my:name[2]:
    value: c
    string: c
    redacted: c
    object: DhAABnN0cmluZwwDAAFj
    secure: false
    isObject: false
  my:name[3]:
    value:
    - nested: value
    string: '[{"nested":"value"}]'
    redacted: '[{"nested":"value"}]'
    object: HRAADltdaW50ZXJmYWNlIHt9/4MCAQL/hAABEAAASP+EKAABF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAAc/4IZAAEGbmVzdGVkBnN0cmluZwwHAAV2YWx1ZQ==
    secure: false
    isObject: true
  my:name[3][0]:
    value:
      nested: value
    string: '{"nested":"value"}'
    redacted: '{"nested":"value"}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAAc/4IZAAEGbmVzdGVkBnN0cmluZwwHAAV2YWx1ZQ==
    secure: false
    isObject: true
  my:name[3][0].nested:
    value: value
    string: value
    redacted: value
    object: EhAABnN0cmluZwwHAAV2YWx1ZQ==
    secure: false
    isObject: false
